examples/rds/aurora-mysql/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "humanitec_application" "app" {
+  id   = var.name
+  name = var.name
+}
+
+module "rds" {
+  source = "../../../humanitec-resource-defs/rds/aurora"
+
+  prefix                 = "${var.name}-"
+  resource_packs_aws_rev = var.resource_packs_aws_rev
+  resource_packs_aws_url = var.resource_packs_aws_url
+
+  access_key = var.access_key
+  secret_key = var.secret_key
+  region     = var.region
+
+  type = "mysql"
+
+  engine         = "aurora-mysql"
+  engine_version = "8.0"
+  group_family   = "aurora-mysql8.0"
+
+  name            = "${var.name}-database"
+  database_name   = "my_database"
+  master_username = "username"
+  master_password = "password"
+
+  vpc                    = var.vpc_id
+  subnets                = var.subnet_ids
+  db_subnet_group_name   = "${var.name}-subnet-group"
+  create_db_subnet_group = true
+  security_group_rules = {
+    ingress = {
+      type                     = "ingress"
+      from_port                = 5432
+      to_port                  = 5432
```

Comment:
Looks like this can be removed https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/blob/master/main.tf#L331-L333 which might be better as the mysql example now is using the postgres ports.

Reply:
Done 👍